src/measure_theory/group/action.lean

```diff
@@ -129,6 +129,18 @@ variables {G} [smul_invariant_measure G α μ]
 @[simp, to_additive] lemma measure_smul_set (s : set α) : μ (c • s) = μ s :=
 ((smul_invariant_measure_tfae G μ).out 0 4).mp ‹_› c s
 
+@[simp, to_additive] lemma measure_preserving_const_smul
+  {M α : Type*} [has_smul M α] {m : measurable_space α} [measurable_space M]
+  [has_measurable_smul M α] (μ : measure α) [smul_invariant_measure M α μ] (g : M) :
+  measure_preserving ((•) g : α → α) μ μ :=
```

Comment:
We have `measure_preserving_smul`. Could you please merge these 2 lemmas? It seems that yours is more general because it has weaker TC assumptions.

Reply:
Thanks, I should have noticed this. Now done.